### DISCUSSION TITLE
Welcome to Times-up-flutter Discussions!

### DISCUSSION THREAD

**Discussion by JordyHers** [JordyHers/Times-up-flutter#40] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JordyHers** [comment]
@JordyHers-org/dev-team Here is a discussion section opened to share more. Some of the contributors will be outside contributors. So it's important that they have the chance to get to share their ideas.

**Comment by mrjamesweb** [comment]
Hello
